src/pybind/cephfs/cephfs.pyx

```diff
@@ -538,15 +539,28 @@ cdef class LibCephFS(object):
             raise make_ex(ret, "error calling ceph_init")
         self.state = "initialized"
 
-    def mount(self):
+    def mount(self, mount_root=None, filesystem_name=None):
         """
         Perform a mount using the path for the root of the mount.
         """
         if self.state == "configuring":
             self.init()
         self.require_state("initialized")
+
+        root = "/" if mount_root is None else mount_root
+        cdef:
+            char *_filesystem_name = filesystem_name
```

Comment:
Got this failure:

````
2018-08-21T15:04:27.245 INFO:teuthology.orchestra.run.smithi080.stderr:CephFS mounting...
2018-08-21T15:04:27.246 INFO:teuthology.orchestra.run.smithi080.stderr:Traceback (most recent call last):
2018-08-21T15:04:27.246 INFO:teuthology.orchestra.run.smithi080.stderr:  File "<string>", line 9, in <module>
2018-08-21T15:04:27.246 INFO:teuthology.orchestra.run.smithi080.stderr:  File "/usr/lib/python3/dist-packages/ceph_volume_client.py", line 479, in connect
2018-08-21T15:04:27.246 INFO:teuthology.orchestra.run.smithi080.stderr:    self.fs.mount()
2018-08-21T15:04:27.246 INFO:teuthology.orchestra.run.smithi080.stderr:  File "cephfs.pyx", line 552, in cephfs.LibCephFS.mount
2018-08-21T15:04:27.247 INFO:teuthology.orchestra.run.smithi080.stderr:TypeError: expected bytes, NoneType found
````

/ceph/teuthology-archive/pdonnell-2018-08-21_02:55:01-fs-wip-pdonnell-testing-20180821.011741-testing-basic-smithi/2930400/teuthology.log

Looks like it needs set to at least an empty string?